game_patch/object/event.cpp

```diff
@@ -339,8 +331,104 @@ ConsoleCommand2 debug_event_msg_cmd{
     }
 };
 
+CodeInjection trigger_activate_linked_objects_activate_event_in_multi_patch{
+    0x004C038C,
+    [] (auto& regs) {
+        if (rf::level.version >= 300) {
+            const rf::Object* const obj = regs.esi;
+            const rf::Event* const event = static_cast<const rf::Event*>(obj);
+            static const std::unordered_set blocked_event_types{{
+                rf::EventType::Load_Level,
+                rf::EventType::Drop_Point_Marker,
+                rf::EventType::Go_Undercover,
+                rf::EventType::Win_PS2_Demo,
+                rf::EventType::Endgame,
+                rf::EventType::Defuse_Nuke,
+                rf::EventType::Play_Video,
+            }};
+            // RF does not allow triggers to activate events in multiplayer.
+            // In AF levels, we allow it, unless it is an event that would be 
+            // problematic in multiplayer. 
+            const rf::EventType event_type = static_cast<rf::EventType>(event->event_type);
```

Comment:
(opinion) great idea, enable all events without any synchronization of the game state between server and client, especially late client. It probably introduces some quirks and even bugs, but who cares